pages/linux/createrepo.md

```diff
@@ -0,0 +1,16 @@
+# createrepo
+
+> Initializes an RPM repository in the given directory, including all XML and SQLite.
```

Comment:
Wondering if it would better to end the sentence with `database` or `files` ? (Thoughts)
```suggestion
> Initializes an RPM repository in the given directory, including all XML and SQLite files.
```